rpc/backend/tracing.go

```diff
@@ -109,7 +109,7 @@ func (b *Backend) TraceTransaction(hash common.Hash, config *evmtypes.TraceConfi
 	}
 
 	if config != nil {
-		traceTxRequest.TraceConfig = config
+		traceTxRequest.TraceConfig = b.convertConfig(config)
 	}
```

Comment:
looks like we don't need this nil check since `convertConfig` handles it internally

Reply:
I'm not sure I follow this.  I just mean that `convertConfig` already has the `nil` check so we're checking if `config != nil`  multiple times right?